docs/advanced.rst

```diff
@@ -44,15 +44,57 @@ exceeds the scope of this documentation.
 As a starting point, the OpenSSL manual pages (especially ca_, req_, x509_, cms_,
 verify_ and config_) and Stefan H. Holek's pki-tutorial_ are useful.
 
-.. _ca: https://www.openssl.org/docs/manmaster/man1/ca.html
-.. _req: https://www.openssl.org/docs/manmaster/man1/req.html
-.. _x509: https://www.openssl.org/docs/manmaster/man1/x509.html
-.. _cms: https://www.openssl.org/docs/manmaster/man1/cms.html
-.. _verify: https://www.openssl.org/docs/manmaster/man1/verify.html
-.. _config: https://www.openssl.org/docs/manmaster/man5/config.html
+.. _ca: https://www.openssl.org/docs/man1.1.1/man1/ca.html
+.. _req: https://www.openssl.org/docs/man1.1.1/man1/req.html
+.. _x509: https://www.openssl.org/docs/man1.1.1/man1/x509.html
+.. _cms: https://www.openssl.org/docs/man1.1.1/man1/cms.html
+.. _verify: https://www.openssl.org/docs/man1.1.1/man1/verify.html
+.. _config: https://www.openssl.org/docs/man1.1.1/man5/config.html
 
 .. _pki-tutorial: https://pki-tutorial.readthedocs.io/
 
+.. _sec-key-usage:
+
+Certificate Key Usage Attributes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default (for backwards compatibility reasons), RAUC does not check the
+certificate's key usage attributes.
+When not using a stand-alone PKI for RAUC, it can be useful to enable checking
+via the ``check-usage`` configuration option to allow only specific
+certificates for bundle signing.
```

Comment:
Is 'signing' or 'installation' meant here?

Reply:
Yes, 'installation' is clearer here.